contentcuration/contentcuration/viewsets/invitation.py

```diff
@@ -140,12 +140,21 @@ def perform_update(self, serializer):
     @action(detail=True, methods=["post"])
     def accept(self, request, pk=None):
```

Comment:
I think this implementation being called `accept()` could be a little confusing because now it retrieves and sets accept or decline for an invitation. Could it be called something like `update()` or `set()` ?

Reply:
Thanks for raising this @LianaHarris360. For purposes of clarity well create two endpoints so we can separate the concerns.
1. `invitation/accept`
2. `invitation/decline`